File: uniffi_bindgen/src/bindings/dart/templates/UInt8Helper.py
class _UniffiConverterUInt8(_UniffiConverterPrimitiveInt) {
    CLASS_NAME = "u8"
    VALUE_MIN = 0
    VALUE_MAX = 2**8

    static read(buf) {
        return buf.read_u8()
    }

    static write_unchecked(value, buf) {
        buf.write_u8(value)
    }
}

File: uniffi_bindgen/src/bindings/dart/templates/UInt8Helper.dart
class _UniffiConverterUInt8 extends _UniffiConverterPrimitiveInt {


    static read(_UniffiRustBufferBuilder buf) {
        return buf.data.cast<Uint8>().value;
    }

    static _UniffiRustBufferBuilder write(value) {
        //buf.data.cast<Int16> = value;
        Pointer<_UniffiRustBuffer> _rustBuffer = calloc<_UniffiRustBuffer >();
        return _rustBuffer.ref.buffer;
    }
}
